algorithms/ex_1.rb

```diff
@@ -0,0 +1,33 @@
+def solution(n)
+ 
+  counter = 0
+  max_value = 0
+
+  if n == 1
+    return counter
+  else 
+
+    binary_arr = n.to_s(2).split("")
```

Comment:
when not using interpolation prefer single quotation marks i.e. `'` instead of `"`